CONTRIBUTING.md

```diff
@@ -0,0 +1,110 @@
+# How to Contribute
+
+*eksctl* is [Mozilla Public License Version 2.0 licenced](LICENSE.MPL-2.0) and accepts contributions via GitHub
+pull requests. This document outlines some of the conventions on development
+workflow, commit message formatting, contact points and other resources to make
+it easier to get your contribution accepted.
+
+We gratefully welcome improvements to documentation as well as to code.
+
+# Certificate of Origin
+
+By contributing to this project you agree to the Developer Certificate of
+Origin (DCO). This document was created by the Linux Kernel community and is a
+simple statement that you, as a contributor, have the legal right to make the
+contribution. No action from you is required, but it's a good idea to see the
+[DCO](DCO) file for details before you start contributing code to Scope.
+
+# Chat
+
+The project uses Slack. Join the [Weave community](https://weaveworks.github.io/community-slack/) Slack workspace and use the [#eksctl](https://weave-community.slack.com/messages/eksctl/) channel.
+
+## Getting Started
+
+- Fork the repository on GitHub
+- Read the [README](README.md) for getting started as a user and learn how/where to ask for help 
+- If you want to contribute as a developer, continue reading this document for further instructions
+- Play with the project, submit bugs, submit pull requests!
+
```

Comment:
Encourage to ask questions on Slack if they get stuck. Maybe add a link to ["good first issues"](https://github.com/weaveworks/eksctl/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)?

Reply:
Good idea, i'll add.